.gitignore

```diff
@@ -5,6 +5,10 @@ tmp/
 *.tmp
 *.bak
 *.log
+######
+## sobre cómo enviar as traducións ####
+traducir.md
```

Comment:
i think this was pushed to the wrong repo ;D

Reply:
sure :(

I have the best intentions to try using git 100% from linux console (terminal), so I've added your dandelion-master as a new remote branch to my local git. I made this change to avoid sending you traducir.md, but something didnt work as spected.

I beg you patience with my poor skills but I contribute to several projects and I really need to get practice to avoid github web interface, make all code changes locally and send it to remotes from terminal. It's way faster and productive.

Of course this is none of your problem, it is mine! I need practice.


Thank you for informing me about this issue and I will try to do it better next time ;)

Reply:
I can help you out if you need some command, but you have to give the file structure and what you exact want to do

Reply:
I can recommend you do this before starting on working on something new for a project:

```
git remote add upstream https://github.com/Diaspora-for-Android/dandelion.git
git fetch upstream
git checkout my_working_branch
git reset --hard upstream/master
..... do modification
git commit -am "text"
git push --set-upstream origin my_working_branch
```
and send a PR from that new branch

Reply:
Thank you!
I will try your suggestion.

but, just one more question... 

"git reset --hard upstream/master" is executed when HEAD is on "ga-translation" branch? so this merges your latest code from upstream/master to this branch; then I translate; then add/commit and finally pushing directly from this branch will send you (to upstream/master) the new code to be reviewed and finally merged (if no errors)?

So what I'm doing following your suggestion is to push DIRECTLY to your "master" branch, not to my "master" on github AND THEN making a pull-request from GUI on github?

So no need to "fork" dandelion on github, just need to init a local repo with your master branch as remote; new local branch "ga-translation"; then make changes on this branch and push them directly to you to merge.

Many thanks for your help.